src/include/ipc/control.h

```diff
@@ -102,7 +101,7 @@ enum sof_ipc_ctrl_cmd {
 struct sof_ipc_ctrl_value_chan {
 	uint32_t channel;	/**< channel map - enum sof_ipc_chmap */
 	uint32_t value;
-} __packed;
+} __attribute__((packed));
```

Comment:
Maybe we don't have to revert this. I'm wondering if it'd be good idea to make use of `-include` flag with header that has `__packed` etc so we can make these files to work like they were in kernel.
@mmaka1 @tlauda @lgirdwood @dbaluta What do you think?

Reply:
The include would be hidden inside the make files, so it would be problematic if some external user app would like to take header "as is". With stdint we can at least assume that everyone has such header.